keygen/src/keygen.rs

```diff
@@ -335,6 +336,13 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                         .multiple(true)
                         .validator(grind_validator_starts_and_ends_with)
                         .help("Saves specified number of keypairs whos public key starts and ends with the indicated perfix and suffix\nExample: --starts-and-ends-with sol:ana:4\nPREFIX and SUFFIX type is Base58\nCOUNT type is u64"),
+                )
+                .arg(
+                    Arg::with_name("num_threads")
+                        .long("num-threads")
+                        .value_name("NUMBER")
+                        .takes_value(true)
+                        .help("Specify the number of grind threads, defaults to the number of CPUs on the machine"),
```

Comment:
If you use `.default_value()` then the ", defaults to the number of CPUs on the machine" text can be removed and the cli help will show the actual default number to the user
```rust
let default_num_cpus = num_cpus::get().to_string();
...
.default_value(&default_num_cpus)
```